Content.Server/Xenoarchaeology/XenoArtifacts/Effects/Components/GasArtifactComponent.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Content.Shared.Atmos;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Server.Xenoarchaeology.XenoArtifacts.Effects.Components;
+
+/// <summary>
+///     Spawn a random gas with random temperature when artifact activated.
+/// </summary>
+[RegisterComponent]
+public class GasArtifactComponent : Component
+{
+    public override string Name => "GasArtifact";
+
+    /// <summary>
+    ///     Gas that will be spawned when artifact activated.
+    ///     If null it will be picked on startup from <see cref="PossibleGases"/>.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("spawnGas")]
+    public Gas? SpawnGas;
+
+    /// <summary>
+    ///     List of possible activation gases to pick on startup.
+    /// </summary>
+    [DataField("possibleGas")]
+    public Gas[] PossibleGases =
+    {
+        Gas.Oxygen,
+        Gas.Plasma,
+        Gas.Nitrogen,
+        Gas.CarbonDioxide,
+        Gas.Tritium
+    };
```

Comment:
why not just every gas?

Reply:
Water vapor with random temperature wouldn't work. Beside, I'm still not sure about Tritium, because how destructive it is.

Reply:
i think trit is fine, it also serves as a funny way to extract tritium